test/specs/reference/tests/05.reference_values.js

```diff
@@ -242,6 +242,70 @@ exports.execute = function (options) {
 
             testTupleValidity(6, values, isHTML);
         });
+        
 
     });
+    
+    describe("Testing for JSON AND JSONB Values", function() {
+        var expectedValues=[{"id":"1001","json_col":true,"jsonb_col":true},
```

Comment:
These should be the formatted values. You are testing the `values` here and not `_data`.